HISTORY.md

```diff
@@ -7,6 +7,7 @@
 * Since 6.14, fix false positive flush/compaction `Status::Corruption` failure when `paranoid_file_checks == true` and range tombstones were written to the compaction output files.
 * Since 6.14, fix a bug that could cause a stalled write to crash with mixed of slowdown and no_slowdown writes (`WriteOptions.no_slowdown=true`).
 * Fixed a bug which causes hang in closing DB when refit level is set in opt build. It was because ContinueBackgroundWork() was called in assert statement which is a no op. It was introduced in 6.14.
+* Perform post-flush updates of in-memory data structures of memtable list in a callback that will be called by the thread writing to MANIFEST file before signaling other threads and releasing db mutex.
```

Comment:
When looking at the release notes, I wasn't able to think of the user-visible bug that this referred to. Only by looking at the PR I learned it's a `Get()` correctness issue. Is it possible to write the note more in terms of what a user would observe?

Reply:
Sure, will submit a PR to address your comment.